tgui/packages/tgui/interfaces/PowerMonitor/Apc.tsx

```diff
@@ -8,7 +8,9 @@
 import { useBackend } from '../../backend';
 import { Box, Chart, LabeledList, Stack, Table, Tooltip } from '../../components';
 import { formatPower } from '../../format';
-import { PowerMonitorApcData, PowerMonitorApcItemData } from './type';
+import { SortDirection } from '../PlayerPanel/constant';
```

Comment:
Probably should pull this out to common or something - maybe common/sorting

Reply:
Done, will need to update #18246 depending on merging order but it won't be a big issue.